docs/graphql/manual/schema/relationships/remote.rst

```diff
@@ -0,0 +1,96 @@
+.. meta::
+   :description: Adding a remote schema relationship with Hasura
+   :keywords: hasura, docs, remote relationship, remote join, remote schema, data federation
+
+.. _add_remote_relationship:
+
+Remote relationships
+====================
+
```

Comment:
Can we include a heading called `Introduction` here? The intro text is already here 🙂